src/components/Dashboard/Dashboard.js

```diff
@@ -0,0 +1,96 @@
+import styled from 'styled-components/macro';
+import home from './Home.svg';
+
+function Dashboard({ letActiveUser }) {
+  return (
+    <Wrapper>
+        <HeaderWrapper>
+            <img src={home} alt="Menu"></img>
+        </HeaderWrapper>
+        <BodyWrapper>
+            <BigHeadline>Guten Abend, <br/><span>{letActiveUser}!</span></BigHeadline>
+            <SmallHeadline>Deine Übersicht</SmallHeadline>
+            <GridWrapper>
+                <FullWidthLink></FullWidthLink>
+                <ChatLink></ChatLink>
+                <ChatLink></ChatLink>
+                <StundenLink></StundenLink>
```

Comment:
Maybe add an aria-label to the link? 
[Using aria-label for link purpose](https://www.w3.org/WAI/WCAG21/Techniques/aria/ARIA8.html)